src/System.Runtime.InteropServices/ref/System.Runtime.InteropServices.cs

```diff
@@ -490,10 +490,12 @@ public static partial class Marshal
         public static string PtrToStringUni(System.IntPtr ptr) { return default(string); }
         [System.Security.SecurityCriticalAttribute]
         public static string PtrToStringUni(System.IntPtr ptr, int len) { return default(string); }
+#if NETCOREAPP11
```

Comment:
The guidance is to use what we are conditioning as the condition, so in this case $(TargetGroup) but it isn't always followed. in the dev/api branch we have a lot of netstandard17 #ifdefs. 

However this is an intermediate commit so I'm not sure it matters unless you plan to not merge the all of these.